doc_source/getting-started-kubernetes.md

```diff
@@ -115,6 +116,28 @@ Replace *account\-id* and *Region\-code* with one of the appropriate sets of val
 856666278305\.dkr\.ecr\.ap\-east\-1\.amazonaws\.com/aws\-appmesh\-envoy:v1\.22\.0\.0\-prod
 422531588944\.dkr\.ecr\.eu\-south\-1\.amazonaws\.com/aws\-appmesh\-envoy:v1\.22\.0\.0\-prod
 924023996002\.dkr\.ecr\.af\-south\-1\.amazonaws\.com/aws\-appmesh\-envoy:v1\.22\.0\.0\-prod
+
+For controller image:
```

Comment:
For older versions point to this link: https://docs.aws.amazon.com/eks/latest/userguide/add-ons-images.html

Mention the versions which the AWS accounts switched from.